__tests__/app.test.js

```diff
@@ -220,3 +220,33 @@ describe("5. GET /api/users", () => {
       });
   });
 });
+
+describe("6. GET /api/reviews (queries)", () => {
+      test("status:200, responds with indicated category (query)", () => {
+    return request(app)
+      .get("/api/reviews?category=dexterity")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeInstanceOf(Array);
+        expect(reviews).toHaveLength(1);
+        expect(reviews).toBeSortedBy("created_at", { descending: true });
```

Comment:
It seems a little odd to assert a sort order on an array you've asserted only has a single element in - can remove if you'd like